proposals/lip-0057.md

```diff
@@ -1344,10 +1345,12 @@ def isPunished(address: Address, height: uint32) -> bool:
 This function computes the set of active validators and their BFT weight based on the input set of eligible validators.
 
 ```python
-def getActiveValidators(validatorsTwoRoundsAgo: list[EligibleValidatorObject], roundNumber: uint32) -> list[ValidatorObject]:
+def getActiveValidators(roundNumber: uint32) -> list[ValidatorObject]:
     initRounds = genesisDataStore.initRounds
     initValidators = genesisDataStore.initValidators
 
+    validatorsTwoRoundsAgo = snapshotStore(nextRound)
```

Comment:
- `nextRound` is not defined,
- with the changed definition of the snapshot store, `validatorsTwoRoundsAgo` is also confusing

Reply:
Good catch.  `nextRound` replaced by `roundNumber` and modified the name of the snapshot validators to `eligibleValidators`.